### DISCUSSION TITLE
Flight doesn't start and autoortho cuts internet connection.

### DISCUSSION THREAD

**Discussion by denizfiratturanli** [kubilus1/autoortho#198] [Q&A]
Hi,
I just installed autoortho for X-Plane 11.55. When I start the program and then start a flight, autoortho starts giving me error messages. After about 3 minutes, it cuts the internet, including all devices connected to the same Wi-Fi.
Here is a video showing this process.
I appreciate your help.

Kind regards,
Deniz Fırat TURANLI

https://www.youtube.com/watch?v=iBtxtKxwqFU

**Comment by SaffronMan1** [comment]
I have exactly the same problem and I have still not found a solution. I hope it gets sorted out soon
